tests/core/gui/test_create_organization.py

```diff
@@ -0,0 +1,248 @@
+async def test_create_organization_previous_clicked(
+    gui, aqtbot, running_backend, qt_thread_gateway, catch_create_org_widget, autoclose_dialog
+):
+
+    gui.config = gui.config.evolve(default_backend_addr=str(running_backend.addr))
+
+    await aqtbot.key_clicks(gui, "n", QtCore.Qt.ControlModifier)
+
+    co_w = await catch_create_org_widget()
+
+    assert co_w
+    assert co_w.user_widget.isVisible() is True
+    assert co_w.device_widget.isVisible() is False
+    assert co_w.button_previous.isVisible() is False
+    assert co_w.button_validate.isEnabled() is False
+
+    await aqtbot.key_clicks(co_w.user_widget.line_edit_user_full_name, "Gordon Freeman")
+    assert co_w.button_validate.isEnabled() is False
+    await aqtbot.key_clicks(co_w.user_widget.line_edit_user_email, "gordon.freeman@blackmesa.com")
+    assert co_w.button_validate.isEnabled() is False
+    await aqtbot.key_clicks(co_w.user_widget.line_edit_org_name, "AnomalousMaterials")
+    assert co_w.button_validate.isEnabled() is False
+    await aqtbot.mouse_click(co_w.user_widget.check_accept_contract, QtCore.Qt.LeftButton)
+    assert co_w.button_validate.isEnabled() is True
+
+    await aqtbot.mouse_click(co_w.button_validate, QtCore.Qt.LeftButton)
+
+    assert co_w.user_widget.isVisible() is False
+    assert co_w.device_widget.isVisible() is True
+    assert co_w.button_previous.isVisible() is True
+    assert co_w.button_validate.isEnabled() is False
+
+    await aqtbot.key_clicks(co_w.device_widget.line_edit_device, "HEV")
+    assert co_w.button_validate.isEnabled() is False
+    await aqtbot.key_clicks(co_w.device_widget.line_edit_password, "nihilanth")
+    assert co_w.button_validate.isEnabled() is False
+    await aqtbot.key_clicks(co_w.device_widget.line_edit_password_check, "nihilanth")
+    assert co_w.button_validate.isEnabled() is True
+
+    await aqtbot.mouse_click(co_w.button_previous, QtCore.Qt.LeftButton)
+
+    assert co_w.user_widget.isVisible() is True
+    assert co_w.device_widget.isVisible() is False
+    assert co_w.button_previous.isVisible() is False
+    assert co_w.button_validate.isEnabled() is True
+
+    assert co_w.user_widget.line_edit_user_full_name.text() == "Gordon Freeman"
+    assert co_w.user_widget.line_edit_user_email.text() == "gordon.freeman@blackmesa.com"
+    assert co_w.user_widget.line_edit_org_name.text() == "AnomalousMaterials"
+    assert co_w.user_widget.check_accept_contract.isChecked() is True
+    assert co_w.button_validate.isEnabled() is True
+
+    await aqtbot.mouse_click(co_w.button_validate, QtCore.Qt.LeftButton)
+    assert co_w.device_widget.line_edit_device.text() == "HEV"
+    assert co_w.device_widget.line_edit_password.text() == "nihilanth"
+    assert co_w.device_widget.line_edit_password_check.text() == "nihilanth"
+
+
+@pytest.mark.gui
+@pytest.mark.trio
+@customize_fixtures(backend_spontaneous_organization_boostrap=True)
+async def test_create_organization_bootstrap_only(
```

Comment:
Shouldn't this be named `test_create_organization_already_used` instead ?

(and do you think we could simplify the test by using `running_backend.backend.organization.bootstrap` to claim the organization instead of doing this trough the gui ?)